doc/_templates/layout.html

```diff
@@ -177,8 +177,11 @@
 </map>
 
 <div style="background-color: white; text-align: left; padding: 10px 10px 15px 15px">
-<a href="{{ pathto('index') }}"><img src="{{
-pathto("_static/logo2.svg", 1) }}" width="540px" border="0" alt="matplotlib"/></a>
+{%- if builder in ('htmlhelp', 'devhelp', 'latex') %}
```

Comment:
Would it be easier to check if `image/svg+xml` is in [`builder.supported_image_types`](http://www.sphinx-doc.org/en/stable/builders.html)?